docs/_docs/themes.md

```diff
@@ -35,13 +48,78 @@ Jekyll will look first to your site's content, before looking to the theme's def
 Refer to your selected theme's documentation and source repository for more information on what files you can override.
 {: .note .info}
 
-To locate theme's files on your computer, run `bundle show` followed by
-the name of the theme's gem, e.g. `bundle show minima` for default Jekyll's
-theme. Then copy the files you want to override, from the returned path to your root folder.
+To locate theme's files on your computer:
+
+1.  Run `bundle show` followed by the name of the theme's gem, e.g., `bundle show minima` for default Jekyll's theme.
+
+    The location of the theme gem is returned. For example, minima is located in `/usr/local/lib/ruby/gems/2.3.0/gems/minima-2.1.0` on a Mac.
+
+2.  Change to the directory's location and open the directory in Finder or Explorer:
+
+    ```
+    cd /usr/local/lib/ruby/gems/2.3.0/gems/minima-2.1.0
+    open .
+    # for Windows, use "explorer ."
+    ```
+
+    A Finder or Explorer window opens showing the theme's files and directories.
+
+    With a clear understanding of the theme's files, you can now override any theme file by creating a similarly named file in your Jekyll site directory.
```

Comment:
Perhaps an example here would be useful. The user may still be wondering what the full path is they need to create in their site.